cjson/PSPBUILD

```diff
@@ -0,0 +1,30 @@
+pkgname=cJSON
```

Comment:
Can you change the name to only have lower case letters?

Reply:
oh I used the original casing so I could use `cd "$pkgname-$pkgver` later on the scripts.

I'll change the pkgname but I'll have to hardcode the bash scripts. (source folder will still be `cJSON-version`)